app/serializers/api/admin/commodities/commodity_csv_serializer.rb

```diff
@@ -0,0 +1,54 @@
+require_relative '../../../concerns/api/v2/csv/csv_serializer'
```

Comment:
We should add a spec for this serializer to be consistent

Reply:
https://github.com/trade-tariff/trade-tariff-backend/blob/HOTT-1462-all-commodities-csv/spec/serializers/api/v2/csv/commodity_serializer_spec.rb

Reply:
done